helix-common/src/main/java/org/apache/helix/SystemPropertyKeys.java

```diff
@@ -84,4 +84,7 @@
       MetadataStoreRoutingConstants.MSDS_SERVER_ENDPOINT_KEY;
 
   public static final String STATEUPDATEUTIL_ERROR_PERSISTENCY_ENABLED = "helix.StateUpdateUtil.errorLog.enabled";
+
+  public static final String TASK_CURRENT_STATE_PATH_DISABLED =
```

Comment:
With this config set up as "disabled", the default behavior would be to turn on task current state. Is this the expected behavior when customers roll out?

Reply:
Yes. When customers roll out a new versions for *participants*, they should expect the behavior to change accordingly. This system property is for an emergency rollback.